tests/acceptance/init-test.js

```diff
@@ -196,6 +198,11 @@ describe('Acceptance: ember init', function () {
   });
 
   it('configurable CI option', async function () {
+
+    if (isDeprecationRemoved(travisDeprecation.until)){
```

Comment:
@bertdeblock Do you have thoughts on the best way to do the test infrastructure for deprecations that have since "broken"?